### PR TITLE
Enable operator developer message to display in the uninstall operator modal...

### PR DESCRIPTION
**Pull request by sg00dwin**
when removing an installed operator subscription from the subscription details page.

Screen Shot 2020-04-02 at 4 32 53 PM

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx [hunk 1]
```diff
@@ -181,17 +181,27 @@ const subscriptionState = (state: SubscriptionState) => {
...
-    callback: () => createUninstallOperatorModal({ k8sKill, k8sGet, k8sPatch, subscription: obj }),
-    accessReview: {
-      group: kind.apiGroup,
-      resource: kind.plural,
-      name: obj.metadata.name,
-      namespace: obj.metadata.namespace,
-      verb: 'delete',
-    },
-  }),
+  (kind, obj, { clusterServiceVersions }) => {
```
Comment (benjaminapetersen) [inline frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx:184]: I dont think the `ResourceKebob` handles a third arg:

```
// kebab.tsx
actions.map((a) => a(kindObj, resource)),
```
https://github.com/openshift/console/blob/a294f781273799c45a6a898605b5f8afcae70248/frontend/public/components/utils/kebab.tsx#L361

Comment (sg00dwin) [reply frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx:184]: @benjaminapetersen  Would that be the cause of this frontend test error?
```
TypeError: Cannot destructure property `clusterServiceVersions` of 'undefined' or 'null'.
      at Array.Object.<anonymous>.menuActions (packages/operator-
lifecycle-manager/src/components/subscription.tsx:108:1)
```

Comment (benjaminapetersen) [reply frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx:184]: Yup, this:
```
(kind, obj, { clusterServiceVersions }) => {
```
says "I get `kind`, `obj`, and a 3rd arg that I'll pluck out `clusterServiceVersion`.  but that 3rd object is never passed, so it is always `null/undefined`, and `null` never has `null.clusterServiceVersion`.

Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx:184]: Right, this won't work for the list view when fix #4916 merges.

Comment (benjaminapetersen) [reply frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx:184]: If you are wanting to pass the third property, then you need to update `kebab.tsx` to support it:

```
// you could update this, if you need to
actions.map((a) => a(kindObj, resource, csv)), 
// but it looks like _.get(obj, 'status.installedCSV'); installedCSV is already part of obj, so you may not need this.
```

**Pull request closed without merging**